frontend/app/routes/$lang/_public/apply/$id/adult-child/applicant-information.tsx

```diff
@@ -259,12 +258,12 @@ export default function ApplyFlowApplicationInformation() {
                 variant="primary"
                 loading={isSubmitting}
                 endIcon={faChevronRight}
+                startIcon={faChevronLeft}
```

Comment:
This `startIcon` should be on the "Back" `ButtonLink` below instead.

Reply:
Good catch! Fixed